keras/backend/theano_backend.py

```diff
@@ -409,6 +409,11 @@ def gather(reference, indices):
 
 # ELEMENT-WISE OPERATIONS
 
+def identity(x):
+    """Returns a tensor with the same shape, type and content as the input tensor.
+    """
+    return x.copy()
```

Comment:
Does this create a copy of the tensor? I suspect it does. We need to just copy the pointer without touching the data in memory.